pages/cnaf/beneficiaire/index.js

```diff
@@ -182,6 +183,7 @@ export default function Beneficiaire() {
                 <td>{r.timestamp}</td>
                 <td>{r.filename}</td>
                 { devMode && <td>{r.fileSize}</td>}
+                { devMode && <td>{!isNaN(r.duration) ? r.duration/1000 : "n/a"}</td>}
```

Comment:
```suggestion
                { devMode && <td>{!isNaN(r.duration) ? r.duration/1000 : "#N.A"}</td>}
```
pour qu'en cas d'import dans Excel ou similaire la valeur importée soit interprétée directement.

Reply:
Oups, c'est `#N/A` la bonne valeur je pense.